.github/workflows/push.yml

```diff
@@ -16,6 +16,21 @@ env:
   TEST_KBC_STORAGE_API_TOKEN: ${{ secrets.TEST_KBC_STORAGE_API_TOKEN }}
   TEST_KBC_STORAGE_API_TOKEN_MASTER: ${{ secrets.TEST_KBC_STORAGE_API_TOKEN_MASTER }}
   TEST_KBC_STORAGE_API_TOKEN_EXPIRED: ${{ secrets.TEST_KBC_STORAGE_API_TOKEN_EXPIRED }}
+
+  # S3 repository
+  IAM_USER: keboola-as-code-s3
```

Comment:
iam user je `keboola-as-code-dist-s3-upload`

Reply:
ok, je to len, ak by ho niekto potreboval najst